test/orm/declarative/test_typed_mapping.py

```diff
@@ -110,6 +111,9 @@ class _SomeDict2(TypedDict):
 _Recursive695_1: TypeAlias = _Recursive695_0
 _Recursive695_2: TypeAlias = _Recursive695_1
 
+_TypingLiteral = typing.Literal["a", "b"]
```

Comment:
**Federico Caselli** (CaselIT) wrote:

same for this and the @testing.requires.python38

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5504

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5504